releasenotes-builder/src/main/java/com/github/checkstyle/CliOptions.java

```diff
@@ -35,6 +35,8 @@
 
     /** Path to a local git repository. */
     private String localRepoPath;
+    /** Path to remote github repository. */
+    private String remoteRepoPath;
```

Comment:
no default value, this will damage main main repo build on travis to generate release notes.
Please update checkstyle/checkstyle travis to let me merge them together.

Reply:
Done.